### DISCUSSION TITLE
Help requested CurrencyInput

### DISCUSSION THREAD

**Discussion by curtic00** [vikramlearning/blazorbootstrap#952] [Q&A]
Blazor Bootstrap V 3.0.0

   AuthorizeView Roles="..." Context="authContext">
      Authorized>      (Note: if I put the starting < in these line, githib won't display the lines.. ????)
         <Tabs @ref="tabs">
         
         </Tabs>

        <Modal CloseOnEscape=false ShowCloseButton=false @ref="modal" Title="Edit Invoice" UseStaticBackdrop=false>
         
                <EditForm EditContext="editContext" @ref="invoiceEditForm" OnValidSubmit="ValidInvoiceSubmit">
                        ........
Offending line:       <CurrencyInput TValue="int" @bind-value="@amount7" Locale="en-GB" Placeholder="Enter amount"  />

    /Authorized>
    NotAuthorized>
       Go away...
    /NotAuthorized>
/AuthorizeView>
......

@Code
....

    private int amount7 = 23;

Error in Edge browser:

blazor.server.js:1  [2024-11-25T16:01:13.957Z] Error: System.ArgumentNullException: Value cannot be null. (Parameter 'accessor')
   at Microsoft.AspNetCore.Components.Forms.FieldIdentifier.Create[TField](Expression`1 accessor)
   at BlazorBootstrap.CurrencyInput`1.OnInitializedAsync()
   at Microsoft.AspNetCore.Components.ComponentBase.RunInitAndSetParametersAsync()

Anyone know what I am doing wrong?

**Comment by gvreddy04** [comment]
@curtic00 Please create a GitHub repository with minimal steps to reproduce the issue. I'll take a look.

**NOTE:** .zip attachments are not allowed.
